first completed iteration of page layout build

Ashley-Ann opened the pull request.

imatari committed:
initial commit of in progress layout build

krittiyaclark reviewed (commented):
No need to merge to the master branch.

imatari committed:
just a test. pls delete junk.txt file

imatari committed:
Create throw-away.txt

imatari committed:
Merge branch 'pageLayout' of https://github.com/krittiyaclark/Build-And-Learn-Budget-App into pageLayout *diss my own stank!*

imatari committed:
Delete throw-away.txt

was just a test to sync to local repo

imatari committed:
Merge branch 'pageLayout' of https://github.com/krittiyaclark/Build-And-Learn-Budget-App into pageLayout

imatari committed:
Delete junk.txt

was a test to try to push from local to remote branch that was not the master branch

imatari committed:
continued building of app

imatari committed:
continued building of page layout for app

imatari committed:
revised page layout: brought form fields to one line. added icon for submit button

imatari committed:
deleted lines that used columns as spacers and used offset instead (thanks Pai!)

imatari commented:
hi Pai! pageLayout is ready for you to review again : ) thanks!
